email-service/pom.xml

```diff
@@ -30,6 +45,11 @@
             <version>6.1.11</version>
```

Comment:
Don't think we need a version and scope for this dependency

Reply:
Deleted spring-web